suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -629,10 +613,28 @@ export const startDescriptorPreloadedDiscoveryThunk = createThunk(
             }),
         );
 
+        // Some cardano derivation are not supported by the device. We need to filter them out.
+        let availableCardanoDerivations: ('normal' | 'legacy' | 'ledger')[] | undefined;
```

Comment:
maybe you can set this to `= []` so you don't need to do `?? []` later.

Reply:
eddc551dae